### PR TITLE
P5 hark input component

### PR DESCRIPTION
**Pull request by HKanwal**
## Description
It works perfectly.

## Related Issue
- [x] This pull request relates to at least one particular issue

  
  - [User Authentication](https://github.com/ps-toronto-team-4/.github/issues/5)

## Checks
- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

## ℹ Additional Information

Jeremie is my idol.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (jere-mie) [inline .vscode/settings.json:1]: let's delete this and add it to the .gitignore

**Review by jere-mie — CHANGES_REQUESTED** [review]
see requested changes

File: .vscode/settings.json (hunk 1 above)
Comment (HKanwal) [reply .vscode/settings.json:1]: done

File: components/Button.tsx [hunk 2]
```diff
@@ -27,7 +27,8 @@ const styles = StyleSheet.create({
         alignItems: 'center',
         justifyContent: 'center',
         width: 250,
-        paddingVertical: 20,
+        height: 50,
+        // paddingVertical: 20,
```
[the remark this answers is not in the document]
Comment (HKanwal) [reply components/Button.tsx]: deleted

File: constants/Layout.ts [hunk 3]
```diff
@@ -9,4 +9,8 @@ export default {
     height,
   },
   isSmallDevice: width < 375,
+  signUpScreen: {
```
[the remark this answers is not in the document]
Comment (HKanwal) [reply constants/Layout.ts]: moved styles into a component.styles.tsx files that is in the same folder as each component.

**Pull request merged**